src/interfaces/resource-interfaces/resource-interface.ts

```diff
@@ -9,6 +9,7 @@ export interface ResourceIF {
   nameRequestType: NameRequestEntityTypes
   addressLabel: string
   filingData: FilingDataIF
+  addressChangeInfo?: string
```

Comment:
Can you move `addressChangeInfo` under `changeData`? I think it's more organized this way.

Reply:
updated

Reply:
This is resolved 👍